cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -1,152 +1,151 @@
+- ../../../../base/operators.coreos.com/subscriptions/nfd
+- ../../../../base/operators.coreos.com/subscriptions/openshift-pipelines-operator-rh
+- ../../../../base/operators.coreos.com/subscriptions/opf-grafana
+- ../../../../base/operators.coreos.com/subscriptions/ovms-operator
+- ../../../../base/operators.coreos.com/subscriptions/pulp-operator
+- ../../../../base/operators.coreos.com/subscriptions/vertical-pod-autoscaler
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/acme-operator
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/cluster-interceptor-opf-ci-pipelines
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/cluster-monitoring-view
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/curator-manager-rb
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/curator-oauth-proxy-rb
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/fybrik-admin-rb
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/moc-nfs-democratic-csi-controller-rb
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/moc-nfs-democratic-csi-node-rb
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/node-labeler
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/scc-privileged-opf-ci-pipelines-aicoe-ci
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/tekton-chains
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/tekton-dashboard
+- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/vault-server-binding
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/acme-operator
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/curator-auth-proxy-role
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/curator-report-editor-role
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/curator-report-manager-role
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/curator-report-viewer-role
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/fybrik-admin-cr
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/moc-nfs-democratic-csi-controller-cr
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/moc-nfs-democratic-csi-node-cr
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/node-labeler
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/tekton-chains
+- ../../../../base/rbac.authorization.k8s.io/clusterroles/tekton-dashboard
+- ../../../../base/storage.k8s.io/csidrivers/org.democratic-csi.nfs
+- ../../../../base/storage.k8s.io/storageclasses/moc-nfs-csi
+- ../../../../base/storage.k8s.io/storageclasses/ocs-external-storagecluster-ceph-rbd
+- ../../../../base/user.openshift.io/groups/cluster-admins
+- ingresscontrollers/default.yaml
+- nodenetworkconfigurationpolicies/vlan-210-nfs.yaml
+- nodenetworkconfigurationpolicies/vlan-211-nese.yaml
+- ../../../../base/core/namespaces/tremor-demo
```

Comment:
whelp, this is a flaw with the notebook, it adds the namespace at the end I think @4n4nd was expecting it to be added alphabetically, we'll need to fix this, for now @GaryPWhite can you remove this line:
`../../../../base/core/namespaces/tremor-demo` (line 135), as it already exists on line 78

Reply:
@GaryPWhite in fact, can  you update this file direclty and add this line, instead of using the notebook? seems like the notebook is reformatting this entire file, hence the large diff, we will try to fix this today, but so as not to block your PR you can just checkout this file: https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml and add the line above alphabetically under `resources` ?

Reply:
sure thing! To be clear -- I ran into issues using the notebook and used the `toolbox` instead. Since the first commit I've been using regular old IDE :)

Reply:
updated!

Reply:
Ah thanks for the info, I think the end result is the same if you're also following the docs, as it's the `kustomize edit add` that seems to reformat the file and append the new lines, we'll have to update both it seems. We appreciate you trying this out as we work through the kinks !

Reply:
no worries! I'd be happy to give more info on the book if that's helpful!

Reply:
(even if that's in the form of opening an issue)

Reply:
yes please, any and all feedback is appreciated !